docs/Contributing.asciidoc

```diff
@@ -226,7 +226,9 @@ Assuming you have already followed steps 1. to 4. above:
 1. Create a separate database: `createdb -O your_username openqa-o3+ where `openqa-o3+ is
    the name you want to use for the database
 2. The next steps must be done by the user you start your local openQA instance with.
-3. Import dump: `pg_restore -c -d openqa path/to/dump`
+3. Import dump: `pg_restore -c -d openqa-o3+ path/to/dump`
```

Comment:
why the "-o3" suffix?

Reply:
I did not introduce the name, I'm just fixing it to agree with what;s being used a couple lines earlier

Reply:
Generally it makes sense to use a suffix, though, assuming you end up with different databases eventually. It's good practice.

Reply:
Yes, I'm usually using a suffix for my DBs like `-o3`/`-osd` for production DBs and e.g. `-some-new-feature` when working on some new feature requiring a DB migration.